vignettes/linking.Rmd

```diff
@@ -15,6 +15,11 @@ require(knitr)
 opts_chunk$set(error=FALSE, message=FALSE, warning=FALSE)
 ```
 
+# Quick setup
```

Comment:
Move this after or inside the Introduction. And no manual line breaks.

Reply:
Done